index.html

```diff
@@ -27,51 +26,36 @@
         <link href="css/theme-lava.css" rel="stylesheet" type="text/css" media="all"/>
         <link href="css/custom.css" rel="stylesheet" type="text/css" media="all"/>
         <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,600,300' rel='stylesheet' type='text/css'>
-
         <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
-
      ga('create', 'UA-17146009-5', 'auto');
      ga('send', 'pageview');
     </script>
     </head>
     <body class="no-loader">
-
         <div class="loader">
             <div class="strip-holder">
                 <div class="strip-1"></div>
                 <div class="strip-2"></div>
                 <div class="strip-3"></div>
             </div>
         </div>
-
         <a id="top"></a>
-
 				<nav class="overlay-nav">
 					<div class="container">
 						<div class="row">
 							<div class="col-md-2">
 								<a target="_self" href="https://fossasia.org"><img alt="FOSSASIA Summit" class="logo logo-light" src="img/fossasia-long.png">
 								<img alt="FOSSASIA" class="logo logo-dark" src="img/fossasia-dark.png"></a>
 							</div>
-
 							<div class="col-md-10 text-right">
 								<ul class="menu">
 									<li><a target="default" class="inner-link" href="#top">Home</a></li>
 									<li><a target="_self" class="inner-link" href="#schedule">Schedule</a></li>
 									<li><a target="_self" href="./exhibitors/index.html">Exhibitors</a></li>
-									<!-- <li><a target="default" class="inner-link" href="#keynotes">Keynotes</a></li>
-									<li><a target="_self" href="./speakers.html">Speakers</a></li>
-                                   <li class="has-dropdown"><a href="./tracks.html">Schedule</a>
-                                        <ul class="nav-dropdown">
```

Comment:
Why removed this ?

Reply:
Because two of them have been used in same block and the line of tags in comments can be regenerated using reference of the similar type of tags for similar function. And you can see this is not useful just holding a bit of space.